lib/stimulus_reflex/reflex.rb

```diff
@@ -153,8 +153,9 @@ def dom_id(record, prefix = nil, hash: "#")
   # morphdom needs content to be wrapped in an element with the same id when children_only: true
   # Oddly, it doesn't matter if the target element is a div! See: https://docs.stimulusreflex.com/appendices/troubleshooting#different-element-type-altogether-who-cares-so-long-as-the-css-selector-matches
   # Used internally to allow automatic partial collection rendering, but also useful to library users
-  # eg. `morph dom_id(@posts), wrap(render(@posts), @posts)`
-  def wrap(content, resource)
+  # eg. `morph dom_id(@posts), render_collection(@posts)`
+  def render_collection(resource, content = nil)
+    content = render(resource) unless content
```

Comment:
Shouldn't we keep `wrap` around for backwards compatibility reasons?

```suggestion
    content = render(resource) unless content
    wrap(content, resource)
  end

  def wrap(content, resource)
```

Reply:
Oh, I just realized `wrap` hasn't even been shipped yet.

Reply:
😁 https://github.com/hopsoft/stimulus_reflex/pull/498/commits/d60a705d55e8af8632e9427f689dfccf8b503407